DQMOffline/Trigger/plugins/PhotonMonitor.cc

```diff
@@ -0,0 +1,302 @@
+      photonPhiME_.numerator->Fill(photons[0].phi());
+      photonEtaME_.numerator->Fill(photons[0].eta());
+      photonVsLS_.numerator -> Fill(ls, photons[0].pt());
+      
+    }
+
+}
+
+void PhotonMonitor::fillHistoPSetDescription(edm::ParameterSetDescription & pset)
+{
+  pset.add<int>   ( "nbins");
+  pset.add<double>( "xmin" );
+  pset.add<double>( "xmax" );
+}
+
+void PhotonMonitor::fillHistoLSPSetDescription(edm::ParameterSetDescription & pset)
+{
+  pset.add<int>   ( "nbins", 2500);
+}
+
+void PhotonMonitor::fillDescriptions(edm::ConfigurationDescriptions & descriptions)
+{
+  edm::ParameterSetDescription desc;
+  desc.add<std::string>  ( "FolderName", "HLT/Photon" );
+  desc.add<edm::InputTag>( "met",      edm::InputTag("pfMet") );
+  desc.add<edm::InputTag>( "jets",     edm::InputTag("ak4PFJetsCHS") );
+  desc.add<edm::InputTag>( "electrons",edm::InputTag("gedGsfElectrons") );
+  desc.add<edm::InputTag>( "photons",edm::InputTag("gedPhotons") );
+  desc.add<std::string>("metSelection", "pt > 0");
+  desc.add<std::string>("jetSelection", "pt > 0");
+  desc.add<std::string>("eleSelection", "pt > 0");
+  desc.add<std::string>("photonSelection", "pt > 145 && eta<1.4442 && hadTowOverEm<0.0597 && full5x5_sigmaIetaIeta()<0.01031 && chargedHadronIso<1.295");
+  //desc.add<std::string>("photonSelection", "pt > 145");
+  desc.add<int>("njets",      0);
+  desc.add<int>("nelectrons", 0);
+  desc.add<int>("nphotons",     0);
+
+  edm::ParameterSetDescription genericTriggerEventPSet;
+  genericTriggerEventPSet.add<bool>("andOr");
+  genericTriggerEventPSet.add<edm::InputTag>("dcsInputTag", edm::InputTag("scalersRawToDigi") );
+  genericTriggerEventPSet.add<std::vector<int> >("dcsPartitions",{});
+  genericTriggerEventPSet.add<bool>("andOrDcs", false);
+  genericTriggerEventPSet.add<bool>("errorReplyDcs", true);
+  genericTriggerEventPSet.add<std::string>("dbLabel","");
+  genericTriggerEventPSet.add<bool>("andOrHlt", true);
+  genericTriggerEventPSet.add<edm::InputTag>("hltInputTag", edm::InputTag("TriggerResults::HLT") );
+  genericTriggerEventPSet.add<std::vector<std::string> >("hltPaths",{});
+  genericTriggerEventPSet.add<std::string>("hltDBKey","");
+  genericTriggerEventPSet.add<bool>("errorReplyHlt",false);
+  genericTriggerEventPSet.add<unsigned int>("verbosityLevel",1);
+
+  desc.add<edm::ParameterSetDescription>("numGenericTriggerEventPSet", genericTriggerEventPSet);
+  desc.add<edm::ParameterSetDescription>("denGenericTriggerEventPSet", genericTriggerEventPSet);
+
+  edm::ParameterSetDescription histoPSet;
+  edm::ParameterSetDescription metPSet;
+  fillHistoPSetDescription(metPSet);
+  histoPSet.add<edm::ParameterSetDescription>("photonPSet", metPSet);
+  std::vector<double> bins = {0.,20.,40.,60.,80.,90.,100.,110.,120.,130.,140.,150.,160.,170.,180.,190.,200.,220.,240.,260.,280.,300.,350.,400.,450.,1000.};
+  histoPSet.add<std::vector<double> >("photonBinning", bins);
```

Comment:
I think you should overwrite this in the python config
taking into account the expected turnON